docs/modules/sql/pages/mapping-to-kafka.adoc

```diff
@@ -28,19 +28,26 @@ To create a mapping to a Kafka topic in SQL, you must use the `CREATE MAPPING` s
 
 - The name of the topic
 - The address of the Kafka broker
+- Client authentication details
 - How to serialize/deserialize the keys and values in messages
 
 ```sql
 CREATE MAPPING my_topic <1>
 TYPE Kafka <2>
 OPTIONS (
-    'bootstrap.servers' = '127.0.0.1:9092' <3>
+    'bootstrap.servers' = '127.0.0.1:9092', <3>
+    'security.protocol'='SASL_SSL', 
+    'sasl.jaas.config'='org.apache.kafka.common.security.plain.PlainLoginModule
+    required username="<my_api_key>"
+    password="<my_api_secret>";',
```

Comment:
Thanks. Only one little thing. There must be a semi column after the password field. Meaning the clause must end with a semi column. You have it in the example, however it is easy to overlook. It might be helpful to emphasize this little detail to the user

Reply:
Thanks @orcunc - I have added a sentence to emphasize the requirement.